test/AppLifecycle/FunctionalTests.cpp

```diff
@@ -35,33 +46,37 @@ namespace ProjectReunionCppTest
         const std::wstring c_testPackageFullName = L"AppLifecycleTestPackage_1.0.0.0_x64__ph1m9x8skttmg";
 
     public:
-        BEGIN_TEST_CLASS(AppLifecycleFunctionalTests)
+        BEGIN_TEST_CLASS(AppLifecycleTests)
+            TEST_CLASS_PROPERTY(L"IsolationLevel", L"Method")
             TEST_CLASS_PROPERTY(L"ThreadingModel", L"MTA")
-            TEST_CLASS_PROPERTY(L"RunFixtureAs:Class", L"InteractiveUser")
+            TEST_CLASS_PROPERTY(L"RunAs:Class", L"RestrictedUser")
         END_TEST_CLASS()
 
         TEST_CLASS_SETUP(ClassInit)
         {
-            // Deploy packaged app to register handler through the manifest.
-            //RunCertUtil(c_testPackageCertFile);
-            //InstallPackage(c_testPackageFile);
```

Comment:
Revert your removal of these commented out tests throughout the file.  They will be enabled very soon now that you have your feature going in.  That is why they are commented out.

Reply:
Restored, though you shouldn't need your own cert anymore given MSTest.cer. We should be able to share 1 dev test cert, not need 1 per feature.

Speaking of...looks like you intend to install AppLifecycleTestPackage.msixbundle. Question: How do you sign that?

You should take a look at how I handled it for DynDep. Hint: DevCheck adds the cert and makes sure there's a test .pfx available to create signed packages during .sln's build (yay inner-loop).

P.S. DevCheck has a couple of refinements coming soon (after this PR) but they're internal refinements, nothing that'll change the developer experience: run DevCheck once, then load and run the solution. And that's true whether 'the developer' is you and I, the CI/Pipeline, Helix, etc